app/src/main/res/layout/activity_tracker.xml

```diff
@@ -100,26 +103,25 @@
             android:layout_height="wrap_content"
             android:layout_gravity="center"
             android:gravity="center"
-            android:text="Add Activity"
+            android:text="@string/add_activity"
             android:textColor="@color/themeColor"
-            android:textSize="18dp"
+            android:textSize="18sp"
             android:textStyle="bold" />
         <TextView
             android:layout_width="200dp"
-            android:textSize="27dp"
+            android:textSize="27sp"
             android:layout_gravity="center"
             android:textStyle="bold"
             android:gravity="center"
             android:layout_margin="10dp"
             android:layout_height="wrap_content"
-            android:text="Your Activities" />
+            android:text="@string/your_activities" />
 
         <ListView
             android:id="@android:id/list"
             android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:drawSelectorOnTop="false"
-            />
+            android:layout_height="375dp"
+            android:drawSelectorOnTop="false" />
```

Comment:
Fixed the error, redesigned the page layout and added the missing strings.
Source:
https://www.youtube.com/watch?v=vXjJyLpydxg
https://www.youtube.com/watch?v=YaWmNhlOXYk

Reply:
Careful when going from relative to fixed sizes.